src/xnnpack/microfnptr.h

```diff
@@ -2107,6 +2107,13 @@ typedef void (*xnn_f32_vbinary_ukernel_fn)(
     float* output,
     const union xnn_f32_default_params params[XNN_RESTRICT XNN_MIN_ELEMENTS(1)]);
 
+typedef void (*xnn_s16_vbinary_ukernel_fn)(
+  size_t batch,
+  const int16_t* input_x,
+  const int16_t* input_y,
+  int16_t* output,
+  const union xnn_s16_cvt_params params[XNN_RESTRICT XNN_MIN_ELEMENTS(1)]);
```

Comment:
qs16_mul params

Reply:
as we are supporting minmax, updated the params as qs16_mul_minmax params